docs/info.md

```diff
@@ -1,69 +1,62 @@
 `apimetrics info`
 =================
 
-Information about APImetrics monitoring infrastructure.
+Information about APImetrics monitoring infrastructure, which can be separated into two sections:
 
-* [`apimetrics info locations`](#apimetrics-info-locations)
-* [`apimetrics info orgs`](#apimetrics-info-orgs)
-* [`apimetrics info regions`](#apimetrics-info-regions)
-* [`apimetrics info whoami`](#apimetrics-info-whoami)
+* [Config](#config)
+  * [`apimetrics info whoami`](#apimetrics-info-whoami)
+  * [`apimetrics info orgs`](#apimetrics-info-orgs)
+* [Agents](#agents)
+  * [`apimetrics info locations`](#apimetrics-info-locations)
+  * [`apimetrics info regions`](#apimetrics-info-regions)
```

Comment:
These are automatically generated so I don't think we can really edit the format